### PR TITLE
Doc-HTTP-Methods

### PR DESCRIPTION
**Pull request by bmckercher123**
New How To from backlog work. Can merge at any time.

Alex reviewed.

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 43d16c56edb69a5d28968912ce24c7beb02e5ff5

https://deploy-preview-2670--calico.netlify.com

File: master/security/http-methods.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: Using HTTP methods and paths in policy rules
+---
+
+### Big picture
+
+Use Calico network policy for Istio-enabled apps to restrict ingress traffic that match HTTP methods or paths.
```
Comment (caseydavenport) [inline master/security/http-methods.md]: "That matches"

File: master/security/http-methods.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+title: Using HTTP methods and paths in policy rules
+---
+
+### Big picture
+
+Use Calico network policy for Istio-enabled apps to restrict ingress traffic that match HTTP methods or paths. 
+
+### Value
+
+Customer-facing applications like web apps can benefit from selective restrictions on ingress traffic (for example, HTTP GET requests).
```
Comment (caseydavenport) [inline master/security/http-methods.md]: Is there no value for east-west (inside the cluster) traffic as well? I feel like this value statement isn't complete.

File: master/security/http-methods.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+### Value
+
+Customer-facing applications like web apps can benefit from selective restrictions on ingress traffic (for example, HTTP GET requests). 
+
+### Features
+
+This how-to guide uses the following Calico features:
+
+- **NetworkPolicy** 
+- **GlobalNetworkPolicy** with http match criteria to restrict traffic using:
```
Comment (caseydavenport) [inline master/security/http-methods.md]: This structure makes it seem like these are only supported on GNP, when actually they are supported on both NP and GNP.

File: master/security/http-methods.md (hunk 1 above)
Comment (bmckercher123) [reply master/security/http-methods.md]: fixed

File: master/security/http-methods.md (hunk 2 above)
Comment (bmckercher123) [reply master/security/http-methods.md]: Better now?

File: master/security/http-methods.md (hunk 3 above)
Comment (bmckercher123) [reply master/security/http-methods.md]: Fixed.

Comment (bmckercher123) [reply master/security/http-methods.md]: Fixed

**Pull request merged**